app/helpers.php

```diff
@@ -28,9 +28,14 @@ function edit_image($image, $coords)
 function incrementTransactionId($request)
 {
     $transactionId = $request->header('X-Request-ID');
-    $transactionIdParts = explode('-', $transactionId);
-    $incrementedStep = $transactionIdParts[1] + 1;
-    $newTransactionId = $transactionIdParts[0] . '-' . $incrementedStep;
 
-    return $newTransactionId;
+    if ($transactionId) {
+        $transactionIdParts = explode('-', $transactionId);
+        $incrementedStep = $transactionIdParts[1] + 1;
+        $newTransactionId = $transactionIdParts[0] . '-' . $incrementedStep;
+
+        return $newTransactionId;
+    }
+
+    return null;
```

Comment:
hmm ya i guess we could create one instead. The reason I did this was mainly for testing. When trying to test the API, the tests were failing cause we don't send over a transaction ID with the fake requests. But I could see an instance where requests are coming in without the header for reals. But in those cases, would we want rogue to be responsible for creating the header here? Or would we want to not accept the request without that header?